### PR TITLE
Update use case examples for 2.4

### PR DESCRIPTION
**Pull request by JieXiong9119**
Update schematron and examples for all use cases to BuildingSync v2.4. The SEED use case will be held until the SEED updates its BuildingSync compatibility to v2.4.

### CODE REVIEW

File: BETTER/src/BETTER_Input-1.0.0.csv [hunk 1]
```diff
@@ -1,7 +1,7 @@
 "phase title","phase see","pattern title","pattern see","rule title","rule context","generate context test","assert test","assert description","assert severity","notes"
 "BETTER Building Information","Based on BETTER analysis inputs","Basic Building Info","","Basic Building Info","/auc:BuildingSync/auc:Facilities/auc:Facility/auc:Sites/auc:Site/auc:Buildings/auc:Building","TRUE","auc:Address or auc:ClimateZoneType or auc:WeatherDataStationID or (auc:Latitude and auc:Longitude)","","ERROR",""
 "","","","","","","","auc:PremisesName","","ERROR",""
-"","","","","","","","auc:eGRIDRegionCode","","ERROR",""
+"","","","","","","","auc:eGRIDSubregionCodes","","ERROR",""
```
Comment (macintoshpie) [inline BETTER/src/BETTER_Input-1.0.0.csv:4]: 💯  thanks for updating the CSV!

File: BETTER/examples/BETTER-1.0.0_SampleOffice.xml [hunk 2]
```diff
@@ -16,10 +16,12 @@
               </auc:PremisesIdentifiers>
               <auc:Address>
                 <auc:City>San Bernardino</auc:City>
-                <auc:State>California</auc:State>
+                <auc:State>CA</auc:State>
```
Comment (macintoshpie) [inline BETTER/examples/BETTER-1.0.0_SampleOffice.xml:19]: Nice fix 👍

**Review by macintoshpie — APPROVED** [review]
Awesome job! That was a lot of updating!

File: BETTER/examples/BETTER-1.0.0_SampleCourthouse.xml [hunk 3]
```diff
@@ -16,10 +16,12 @@
               </auc:PremisesIdentifiers>
               <auc:Address>
                 <auc:City>Banning</auc:City>
-                <auc:State>California</auc:State>
+                <auc:State>CA</auc:State>
```
Comment (nllong) [inline BETTER/examples/BETTER-1.0.0_SampleCourthouse.xml:19]: nice catch

**Review by nllong — APPROVED** [review]
sweet!

**Pull request merged**